daisy_workflows/styleguide.md

```diff
@@ -0,0 +1,61 @@
+# Daisy Workflow Styleguide
+Recommendations from the GCE team about how to style daisy workflows.
+
+## Workflow file naming
+Use `.wf.json` for workflow names and prefer underscores in file names.
+
+For example: `my_awesome_build.wf.json`.
+
+## Workflow naming
+Use dashed names for workflow names. `Name": "my-awesome-build"`
+
+## Step naming
+Don't use camel case for step naming as this looks too much like the Steps
```

Comment:
maybe make this text similar to above:
+Use dashed names for workflow names: `create-build-disk`, don't use camel case for step naming as this looks too much like the Steps themselves.